### DISCUSSION TITLE
SideDownload Error Zenva Academy Video

### DISCUSSION THREAD

**Discussion by mathie-nav** [aclap-dev/video-downloadhelper#128] [Help and Q&A]
### Your question

Greetings to the community, I would like to know if someone can help me when trying to download videos from a Zenva Academy course page, when trying to download a chapter in any of the definitions I get the error that I show in the images and the text that is returned is the next...

My information:

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-AR
license: accepted
key (only 16 first characters): a20049d32a5d4b28
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "i686",
    "node": "10"
  },
  "home": "C:\\Users\\elDrunkenMonkey",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

> SideDownload error:
[dash @ 0701b8c0] Error when loading first fragment of playlist
[in#0 @ 0701b740] Error opening input: Invalid data found when processing input
Error opening input file https://zavideoplatform.streaming.mediaservices.windows.net///80186611-e40e-47de-ac51-dbef1a381230/Introduction.ism/manifest(format=mpd-time-csf,encryption=cbc).
Error opening input files: Invalid data found when processing input

![chrome_0tr4p81blZ](https://github.com/aclap-dev/video-downloadhelper/assets/132364517/075cea25-fb93-4b9f-9bdd-d456bf40761c)
![yjlRueYAGA](https://github.com/aclap-dev/video-downloadhelper/assets/132364517/fc0bf241-d98a-4def-b64b-036701f44864)

**Comment by GrampaWildWilly** [comment]
You need to read this https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help & follow the instructions there. Once you have followed those instructions, somebody should be able to give you meaningful help. You can't expect much help with what you've said so far. It is inadequate. Follow the instructions & you're more likely to get proper help. On this forum, very much NOT like our old forum on Google, you can edit your own posts. Look above your post & to the right. You'll see 3 dots. Click that. You should be able to take it from there.

**Comment by GrampaWildWilly** [comment]
@paulrouget This one is pretty much a duplicate of #113.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8127ff95-2bcb-465b-ae13-939c88387d38)
[ffprobe.txt](https://github.com/aclap-dev/video-downloadhelper/files/14317727/ffprobe.txt)
[Zenva mp4 Log.txt](https://github.com/aclap-dev/video-downloadhelper/files/14317728/Zenva.mp4.Log.txt)
